app/pages/help.html

```diff
@@ -131,6 +131,8 @@ <h3 id=":reload">:reload</h3>
             <li>Finally run the <span class="command-block">:reload</span> command to run all viebrc commands</li>
         </ul>
         Of course it's also possible to make a mapping or custom command to do this for you.
```

Comment:
I would add a sentence here that leads into the next command, that mentions the restart command. Because the reload command describes how to achieve reloading to startup state without an actual restart, it would be nice to mention that it's not hard to restart Vieb as there is a command to do so.

Reply:
I've added a link to the `:restart` command in the middle instead.  That sentence already mentioned restarting Vieb, and it seems to fit better there.

Reply:
Even better, thanks for fixing this.